frontend/src/components/content/ContentPreviewPopup.tsx

```
@@ -18,75 +22,152 @@ interface ContentPreviewPopupProps {
+    setIsLoading(true);
+    try {
+      console.log('Fetching related content for:', content.uid);
+      console.log('API URL:', `${apiURL}/content/related/${content.uid}${userUID ? `?userId=${userUID}` : ''}`);
+
+      const response = await axios.get(
+        `${apiURL}/content/related/${content.uid}${userUID ? `?userId=${userUID}` : ''}`
+      );
+
+      console.log('Related content response:', response.data);
+
+      if (response.data && response.data.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)